$OpenBSD: patch-src_main_C,v 1.5 2011/07/05 15:08:49 dcoppa Exp $

Fix a bug causing artifacts at the bottom of the screen when using
rxvt-unicode without hints
(From: https://awesome.naquadah.org/wiki/Urxvt_Hints)

--- src/main.C.orig	Mon May  2 11:42:28 2011
+++ src/main.C	Mon May 23 16:34:12 2011
@@ -708,6 +708,8 @@ rxvt_term::window_calc (unsigned int newwidth, unsigne
 
   ncol = width  / fwidth;
   nrow = height / fheight;
+  width = ncol * fwidth;
+  height = nrow * fheight;
 }
 
 /*----------------------------------------------------------------------*/
